crates/pow/README.md

```diff
@@ -22,9 +22,9 @@ The `PoWRateLimiterConfig` struct allows you to customize the behavior of the ra
 
 - **`challenge_lifetime`**: How long a challenge remains valid after generation. After this duration, challenges expire and cannot be submitted. Choose based on expected solving time and security requirements.
 
-- **`growth_rate`**: Controls how aggressively difficulty increases with more active challenges. The difficulty formula is `num_active_challenges << growth_rate`. Higher values mean more aggressive rate limiting. Typical values: 1-3.
+- **`growth_rate`**: Controls how aggressively difficulty increases with more active challenges. The number of active challenges gets multiplied by the growth rate to compute the load difficulty, so higher values mean more aggressive rate limiting.
 
-- **`baseline`**: Sets the initial difficulty baseline. The maximum target is calculated as `u64::MAX >> baseline`. Higher baseline values make challenges harder from the start. Range: 0-63.
+- **`baseline`**: Sets the initial difficulty baseline. Higher baseline values make challenges harder from the start. Range: 0-63.
```

Comment:
Maybe not for this PR: but I'd limit the range here to 0 - 32. A baseline with 32 bits of difficulty would probably already be out of reach in most practical settings.